src/shared/scrapers/TW/index.js

```diff
@@ -0,0 +1,114 @@
+import assert from 'assert';
+import * as fetch from '../../lib/fetch/index.js';
+import * as parse from '../../lib/parse.js';
+import datetime from '../../lib/datetime/index.js';
+
+const scraper = {
+  country: 'iso1:TW',
+  level: 'country',
+  aggregate: 'country',
+  url: 'https://covid19dashboard.cdc.gov.tw/dash3',
+  timeseries: false,
+  sources: [
+    {
+      name: 'Taiwan CDC',
+      url: 'https://sites.google.com/cdc.gov.tw/2019-ncov/taiwan',
+      description: ''
+    }
+  ],
+  certValidation: false,
+  type: 'json',
+  scraper: {
+    '0': async function() {
```

Comment:
NaBD: No need to add this `'0'` layer until we have different ones for different dates.

Reply:
Aha!Nice to know. Thanks.